sdk/keyvault/keyvault-admin/CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Release History
 
-## 4.2.3 (Unreleased)
+## 4.3.1 (Unreleased)
```

Comment:
Usually you don't have to add a section since bot will submit a PR to bump the version/update CHANGELOG etc.

Reply:
I released from the branch already and the bot made (and merged) its PR against the branch -- these changes are coming from the bot already.